### ISSUE TITLE
Vacs de 0min

### ISSUE THREAD

**Issue by pzia** [v-l-m/vlm#403]
**Reported by paparazzia on 11 Sep 2010 18:20 UTC**
Ce ticket est l pour quelques temps, mais le fonctionnement du moteur permettrait de prendre en compte pendant une vac un changement de cap d'un bateau par l'insertion d'une nouvelle position.

Le seul hic avec ce mode de fonctionnement est la restitution  l'utilisateur de sa position "courante" en calculant une pseudo vac au moment de l'affichage de la carte.

des ides l dessus ?

**Comment by pzia** [comment]
**Commented by spf on 21 Sep 2010 20:50 UTC**
C'est quoi exactement le use-case ?

**Comment by pzia** [comment]
**Commented by paparazzia on 22 Sep 2010 05:11 UTC**
This is not a bug but a feature request ;)

Le truc intressant est de conserver un moteur avec vac  5min ce qui suffit pour la plupart des cas, mais de permettre de "dpasser un cap" par exemple, en permettant de forcer une position courante (calcule depuis la dernire vac). D'un point de vue moteur, il y a tout ce qu'il faut je pense, c'est d'avantage une question d'ihm... (pas faisable simplement avant un affichage en canvas)

Bon, aprs c'est un truc  discuter ;)

**Comment by pzia** [comment]
A voir si moteur v2 de @ylafon

**Comment by maitaivw** [comment]
Pas compris l'idee... Le joueur pourrait forcer un run moteur VLM pour son bateau entre deux vacs c'est ca?

**Comment by pzia** [comment]
oui, c'est exactement ça. Mais ça achoppe sur notre ihm trop "agée" principalement (car c'est un mode "expert" qu'il faudrait que le joueur active.

**Comment by maitaivw** [comment]
Ca a l'air bien mais ca pose des questions, entre autre celle du cas ou le fait d'activer le moteur au bon moment fait profiter de qques chouilles de noeud de vent en plus que personne d'autre n'aura (par exemple le vent fait 10nds au debut et a la fin de la vac mais 11nds au milieu).

J'entends deja P&M ;)

Ca me fait penser au sujet du traitement sequentiel des joueurs par le moteur, qui prend la date et heure de traitement au lieu d'une date/heure commune pour tout le monde. Je n'ai pas trouve de ticket pour ca?

**Comment by Phille** [comment]
+1 avec maitai concernant l'invitation de P&M

**Comment by pzia** [comment]
De toute manière je me suis des-assigné du ticket car je n'ai pas l'intention d'y travailler sans un nouveau moteur. 
Après sur le fond, Vlm devrait faire des vacs en continu ce qui serait plus scalable et enleverait des discussions (et m'eviterait de devoir vous reexpliquer pourquoi le fonctionnement de VLM est le bon quand il prends le temps réel pour faire son calcul). 

Ah, voilà p&m, je sors.

**Comment by Phille** [comment]
Cool, c'est pour quand le nouveau moteur?  ;)

**Comment by maitaivw** [comment]
Je veux bien que tu re-expliques qd meme pourquoi c'est mieux de traiter chaque bateau VLM a la meme position geographique avec un vent different :)
